ledger/sandbox/src/main/resources/db/migration/h2database/V13__Events_new_schema.sql

```diff
@@ -38,7 +38,8 @@ create table participant_events
 );
 
 -- support ordering by offset and transaction, ready for serving via the Ledger API
-create index on participant_events(event_offset, transaction_id, node_index);
+create index participant_events_event_offset_transaction_id_node_index_idx
+    on participant_events(event_offset, transaction_id, node_index);
```

Comment:
Important! You cannot change past SQL migrations! In practice it's hardly a problem but In theory there's nothing preventing a user from using a persistent H2 backend for the sandbox (and while not strictly necessary I'd prefer us having discipline in this regard, given how fragile migrations are). I understand that unfortunately we didn't do a good job so far in this regard, not assigning proper names to indexes, but I would recommend to drop and re-create the table (possibly giving the index a name that we can refer to in the future). In the future we will probably want to look into dropping migrations entirely for the H2 sandbox, as mentioned in another comment.

Reply:
@stefanobaghino-da 
that is cool... I was going to ask about this... but how to I find the name of the index... and how do I guarantee that the default naming convention does not change with the version upgrade?